Dockerfile.alpine

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.3-alpine3.10@sha256:40278d43a27b6e0563fcc4dd52c991a25741b1a775402aea342ab9a80158e69e as builder
+FROM golang:1.13.3-alpine3.10@sha256:f98761ca3b0855d554085eac8a55e6162f859b3f409c68fa4a7e3302c9d8bf5f as builder
```

Comment:
Would be something like this
`FROM golang:1.14.1-alpine3.11@sha256:xx as builder`